x-pack/legacy/plugins/siem/server/lib/pinned_event/saved_object.ts

```diff
@@ -116,7 +116,7 @@ export class PinnedEvent {
                       pickSavedTimeline(null, {}, request[internalFrameworkRequest].auth || null)
                     )
                 );
-                timelineId = timelineResult.savedObjectId;
+                timelineId = timelineResult.savedObjectId; // eslint-disable-line no-param-reassign
```

Comment:
This makes it really clear that we are mutating the input parameters 👍 which event left as is, to me, a great comment that now devs maintaining it are going to know it is being mutated.